### DISCUSSION TITLE
Could not load file in library : incorrect character encoding

### DISCUSSION THREAD

**Discussion by jimmyfi** [TumblThreeApp/TumblThree#333] [Q&A (Question & Answer)]
Hi
I am now getting this error and i am unable download any blogs. This began happening after 1.6.4 and even going back to older version of tumbltree didnt fix the issue.. could it be that 1.6.4 did something to my files?

**Comment by thomas694** [comment]
Hi,
one of the files has changed for whatever reason.
Can you please have a lock at the log file (normally %LOCALAPPDATA%\TumblThree\TumblThree.log), if you see any errors or hints at the time you get the error message?

**Comment by jimmyfi** [comment]
Hi Log looks like the following

20210715 07.38.08.931	Inf 	IsLongPathSupported: False
20210715 07.38.14.894	Err 	ManagerController.LoadDataBasesAsync: System.Xml.XmlException: Virheellinen tavukoodaus.
   kohteessa System.Runtime.Serialization.Json.JsonEncodingStreamWrapper.ReadEncoding(Byte b1, Byte b2)
   kohteessa System.Runtime.Serialization.Json.JsonEncodingStreamWrapper.ReadEncoding()
   kohteessa System.Runtime.Serialization.Json.JsonEncodingStreamWrapper.InitForReading(Stream inputStream, Encoding expectedEncoding)
   kohteessa System.Runtime.Serialization.Json.XmlJsonReader.SetInput(Stream stream, Encoding encoding, XmlDictionaryReaderQuotas quotas, OnXmlDictionaryReaderClose onClose)
   kohteessa System.Runtime.Serialization.Json.DataContractJsonSerializer.ReadObject(Stream stream)
   kohteessa TumblThree.Domain.Models.Files.Files.LoadCore(String fileLocation) tiedostossa C:\projects\Tumblthree\src\TumblThree\TumblThree.Domain\Models\Files\Files.cs:rivillä 121
   kohteessa TumblThree.Domain.Models.Files.Files.Load(String fileLocation) tiedostossa C:\projects\Tumblthree\src\TumblThree\TumblThree.Domain\Models\Files\Files.cs:rivillä 110
   kohteessa TumblThree.Applications.Controllers.ManagerController.GetIFilesCore(String directory) tiedostossa C:\projects\Tumblthree\src\TumblThree\TumblThree.Applications\Controllers\ManagerController.cs:rivillä 359
   kohteessa System.Threading.Tasks.Task`1.InnerInvoke()
   kohteessa System.Threading.Tasks.Task.Execute()
--- Pinon jäljityksen loppu aiemmasta sijainnista, jossa palautettiin poikkeus ---
   kohteessa System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   kohteessa System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   kohteessa TumblThree.Applications.Controllers.ManagerController.<LoadAllDatabasesAsync>d__62.MoveNext() tiedostossa C:\projects\Tumblthree\src\TumblThree\TumblThree.Applications\Controllers\ManagerController.cs:rivillä 308
--- Pinon jäljityksen loppu aiemmasta sijainnista, jossa palautettiin poikkeus ---
   kohteessa System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   kohteessa System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   kohteessa TumblThree.Applications.Controllers.ManagerController.<LoadDataBasesAsync>d__58.MoveNext() tiedostossa C:\projects\Tumblthree\src\TumblThree\TumblThree.Applications\Controllers\ManagerController.cs:rivillä 207

WIth Kind Regards jimmyfi

**Comment by thomas694** [comment]
Hi,
could it be that you opened one of the files in the Index folder in another program / text editor even if you did not (knowingly) save it there? At least one of the files has a UTF/Unicode format now which is not supported.

To find the file that makes the problems you can close TumblThree and move the files from the Index folder to a backup location. Then move each blog's files (e.g. abc.tumblr and abc_files.tumblr) back to the Index folder and start TumblThree afterwards. Close it again and repeat the step until TumblThree no longer loads the files, the last one added is the broken one.
If you have many blogs, you can use small groups (e.g. 5 or 10 at once) to first narrow it down.

**Comment by jimmyfi** [comment]
Hi. It was that one of the x.tumblr had wrong transcoding and when I just removed it, all began to work again. Thanks  :) Random issues <3
